docs/README.md

```diff
@@ -24,3 +24,36 @@ cd docs && make html
 5. Now you can see the generated html `index.html` under `build/html/`
 
 6. Send PR
+
+# Generating Doxygen docs for Tensor Comprehensions
+
+1. Install Doxygen
+
+Run the command
+
+```Shell
+$ sudo apt-get install doxygen
+```
+
+If you have CentOS machine, the command is `sudo yum install doxygen`
```

Comment:
other instructions don't seem to have CentOs versions

Reply:
generating docs doesn't require centos/linux specificity. So I added it here since I have both machines centos and linux